trakt/users.py

```diff
@@ -480,6 +480,27 @@ def get_stats(self):
         data = yield 'users/{user}/stats'.format(user=slugify(self.username))
         yield data
 
+    @get
+    def get_liked_lists(self, list_type=None, limit=None):
+        """Get items a user likes.
+
+        This will return an array of standard media objects.
+        You can optionally limit the type of results to return.
+
+        list_name possible values are "comments", "lists".
```

Comment:
should this be `list_type` instead of `list_name`?

Reply:
anything that is not reserved word like `list` is ok :)

Reply:
3273a76